data/schema/units/single.cfg

```diff
@@ -8,6 +8,8 @@
 	{SIMPLE_KEY id string}
 	{SIMPLE_KEY gender gender,subst}
 	{SIMPLE_KEY name t_string}
+	{SIMPLE_KEY male_name t_string}
+	{SIMPLE_KEY female_name t_string}
```

Comment:
@CelticMinstrel Are we ending up with two slightly different `[unit]` tags here? As in, there's a list of attributes which the results of `[store_unit]` could create, including `name` but not these two; and there's a superset which is the keys recognised when using `[unit]` to create a unit?

Reply:
I'm not sure that the description "two slightly different `[unit]` tags" fits, but you're correct: the keys accepted in `[unit]` are a superset of the keys actually serialized in a unit. That's not new – the placement keys are never serialized either, for example.

Reply:
In that case, it looks good to me and the Flatpak build-failure isn't a blocker.